mypy/build.py

```diff
@@ -1183,6 +1184,7 @@ def compute_hash(text: str) -> str:
 
 
 def write_cache(id: str, path: str, tree: MypyFile,
+                fine_grained_deps: Dict[str, List[str]],
```

Comment:
I'd rename this to `serialized_fine_grained_deps` just to clarify that it does not have the same type as `State.fine_grained_deps` (List vs. Set).